tests/kuttl/tests/run_simple_playbook_debug/01-assert.yaml

```diff
@@ -0,0 +1,57 @@
+#
+# Check for:
+#
+# - 1 OpenStackAnsibleEE CR
+# - 1 Ansibleee-play pod
+# - 1 Ansibleee-play job
+# - Correct output from ansible play
+#
+apiVersion: ansibleee.openstack.org/v1alpha1
+kind: OpenStackAnsibleEE
+metadata:
+  name: ansibleee-play-debug
+  namespace: openstack
+spec:
+  image: quay.io/openstack-k8s-operators/openstack-ansibleee-runner:latest
+  name: openstackansibleee-debug
+  play: |
+    - name: Print hello world
+      hosts: localhost
+      tasks:
+      - name: Using debug statement
+        ansible.builtin.debug:
+          msg: "Hello, world this is ansibleee-play-debug.yaml"
+status:
+  JobStatus: Active
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  namespace: openstack
+status:
+  phase: Active
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  labels:
+    app: openstackansibleee-debug
```

Comment:
I'm testing this locally and the value of app is `openstackansibleee` in my environment

Reply:
I'm setting it to `ansibleee-play-debug` in the 01-run-debug.yaml. I don't understand why it wouldn't show up like it.

Reply:
I understand. Looks like it can't be easily overridden.